src/julia_internal.h

```diff
@@ -264,10 +265,63 @@ void jl_set_base_ctx(char *__stk);
 void jl_init_threading(void);
 void jl_start_threads(void);
 void jl_shutdown_threading(void);
+
+// Whether the GC is running
+extern char *jl_safepoint_pages;
+static inline int jl_addr_is_safepoint(uintptr_t addr)
```

Comment:
What configuration doesn't support `static inline`? It seems that MSVC should be fine with it in c++ mode?